### PR TITLE
Add ...-now install target to nobleo get

### PR DESCRIPTION
**Pull request by Timple**

### CODE REVIEW

File: installer/parse-install-yaml.py [hunk 1]
```diff
@@ -76,9 +76,8 @@ def main():
                 if "version" in install_item:
                     command += " {0}".format(install_item["version"])
 
-            elif install_type == "target" or install_type == "system" or install_type == "pip" or \
-                    install_type == "pip2" or install_type == "pip3" or install_type == "ppa" or \
-                    install_type == "snap" or install_type == "dpkg":
+            elif install_type in ["target", "system", "pip", "pip2", "pip3", "ppa", "snap", "dpkg",
+                                  "system-now", "pip-now", "pip2-now", "pip3-now", "ppa-now", "snap-now"]:
```
Comment (ar13pit) [inline installer/parse-install-yaml.py:80]: Haha! This is a nice hack :)

**Comment by ar13pit** [comment]
> I think we need to update the readme as well.

Yeah. I'll do that and then merge this.

**Comment by MatthijsBurgh** [comment]
I think the README can be more elegant. As we already have a table of all the functions. So Explaining them twice, is not that nice.

**Comment by ar13pit** [comment]
> I think the README can be more elegant. As we already have a table of all the functions. So Explaining them twice, is not that nice.

The two things are complementary as I have not repeated everything from the table. So I think its fine, but if you can make it better then make a PR :)

File: README.md [hunk 2]
```diff
@@ -119,6 +119,17 @@ Depending on Ubuntu distribution:
...
+#### (System / PIP / PIP2 / PIP3 / PPA / Snap)-now
+The default installation method for targets of type `system`, `pip(2/3)`, `ppa` and `snap` is to collect all such targets in a list and install them simultaneously at the end of the `tue-get install` procedure. To install such a dependency immediately for a specific target, use the target type as `X-now`:
+```yaml
+- type: [system/pip/pip2/pip3/ppa/snap]-now
+  name: <Name of the candidate>
+  
+- type: [target/system/pip/pip2/pip3/ppa/snap/dpkg]
+  name: <Name of the candidate>
+```
+or use `tue-install-X-now` functions in `install.bash` along with the procedure to installing the required target.
```
Comment (Timple) [inline README.md:131]: Actually this PR makes sure exactly this doesn't have to happen anymore :clown_face:

Comment (ar13pit) [reply README.md:131]: True. But it is still allowed to do so with `install.bash`. Maybe I should add that it is not recommended to do so with `install.bash` anymore?

Comment (Timple) [reply README.md:131]: Something like that would be nice.

**Pull request merged**